config/jobs/image-pushing/README.md

```diff
@@ -160,7 +160,7 @@ If you need the `.git` directory to be uploaded to your build environment:
 - add an empty `.gcloudignore` file to your repository. This will override the
   [default values][gcloudignore].
 
-[gcr instructions]: https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/README.md
+[gcr instructions]: https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/README.md
```

Comment:
the instructions are still at k8s.gcr.io

ref https://github.com/kubernetes/test-infra/pull/28674#discussion_r1105059589